statistics-production/ees.qmd

```diff
@@ -1240,7 +1264,12 @@ Once the release has been marked as approved, go to the  ‘Pre-release access
 
 ![](../images/EES-invite-pra.PNG)
```

Comment:
this .PNG is caps - will it cause issues?

Reply:
Nope - it just needs to match what is in the folder